src/pid/issuing.ts

```diff
@@ -135,14 +149,55 @@ const getPar =
     );
   };
 
+/**
+ * Make an authorization request
+ */
+const getAuthenticationRequest =
+  ({ appFetch = fetch }: { appFetch?: GlobalFetch["fetch"] }) =>
+  async (
+    clientId: string,
+    requestUri: string,
+    pidProviderEntityConfiguration: CredentialIssuerEntityConfiguration,
+    cieData: CieData
+  ): Promise<AuthenticationRequestResponse> => {
+    const authzRequestEndpoint =
+      pidProviderEntityConfiguration.payload.metadata.openid_credential_issuer
+        .authorization_endpoint;
+
+    const params = new URLSearchParams({
+      client_id: clientId,
+      request_uri: requestUri,
+      name: cieData.name,
+      surname: cieData.surname,
+      birth_date: cieData.birthDate,
+      fiscal_code: cieData.fiscalCode,
+    });
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question(privacy):** It's ok to pass user's personal data as plain query string? 

Theoretically speaking, it's safe to pass sensitive data as query string as long as the connection is over SSL (can we enforce that?). Even though, we still have the problem of urls being logged by web servers and proxies.  

Maybe it's not a concern for this PR but for the protocol itself, maybe it has been already discussed and it's fine. But I'd rather double-check the decision and make sure we have all the needed countermeasures in action.

Reply:
Where is the sensitive data? The data that is passed now is identifying data only. However, this is a workaround while waiting for authentication with "Enter with CIE" to be implemented, where the identification data will be obtained directly from the service provider.

Reply:
Ok my bad, I thought we were passing them here as well

Reply:
Although it's perfectly fair for the current status of the project, given the importance and visibility of the initiative I would write an extensive comment on how this is a temporary workaround.
Hopefully, this source will drive the interest of the citizens, and someone may make the same assumption I did. 

For example
```ts
// User's personal data is not supposed to transit in this flow,
// but to be provided to the PID issuer directly by its chosen authentication method (CIE).
// Being the project in an initial phase, and being we were still unable to fully comply with authentication,
// we temporarily provide data from the App's logged user.
```